dask_sql/cmd.py

```diff
@@ -32,17 +41,80 @@ class CompatiblePromptSession:
     """
 
     def __init__(self, lexer) -> None:  # pragma: no cover
+        kwargs = {
+            "lexer": lexer,
+            "history": FileHistory("dask-sql-history"),
+            "auto_suggest": AutoSuggestFromHistory(),
+            "completer": meta_command_completer,
+        }
         try:
             # Version >= 2.0.1: we can use the session object
             from prompt_toolkit import PromptSession
 
-            session = PromptSession(lexer=lexer)
+            session = PromptSession(**kwargs)
             self.prompt = session.prompt
         except ImportError:
             # Version < 2.0: there is no session object
             from prompt_toolkit.shortcuts import prompt
 
-            self.prompt = partial(prompt, lexer=lexer)
+            self.prompt = partial(prompt, **kwargs)
+
+
+def display_markdown(content, **kwargs):
+    df = pd.DataFrame(content, **kwargs)
+    print(df.to_markdown(tablefmt="fancy_grid"))
+    return True
+
+
+def parse_meta_command(sql):
+    command, _, arg = sql.partition(" ")
+    command = command.strip().replace("+", "")
```

Comment:
I am using `pgcli` to make the meta-commands, pgcli has support for specifying +, (if specified it will show some extra information like Description and size, etc ) so thought some people use the same notion as `\dt+` etc. I have removed that now !!!